src/ui/components/Card/styles.scss

```diff
@@ -66,6 +66,9 @@ $footer-padding: 10px;
     font-weight: normal;
     padding: $footer-padding;
     text-decoration: none;
+    overflow: hidden;
+    text-overflow: ellipsis;
```

Comment:
This seems like a good idea, but I want to check with our UX folks to see what they think about possibly shortening the text of the link using an ellipsis. @aaronrbenson Do you see any issues with this approach?

Reply:
About this approach I didn't wanted to manage this issue that should be a "forced" situation.
If the Card-header is very long (more than 50% of the parent) it may overlaps. This happens only if Card-footer-link is already at 50% already cropped by the ellipsis.

I didn't want to change too much code at my first PR here. 
Do you think I should put the ellipsis to the Card-header too?

![image](https://user-images.githubusercontent.com/17780574/69887656-55d5a980-12e8-11ea-902b-f733e445886c.png)

Reply:
Thanks for the update @fantonangeli. I think the likelihood of the card header being too wide is less than that for the footer, but it's a good point you raise.

@aaronrbenson Could you comment on whether you think it's okay to truncate the text of the card footer and/or header via an ellipsis in order to avoid them overlapping one another?

Reply:
@brassy- Here's the question I asked you about. I just want to check that we're okay with possibly truncating the two headings at the top of the card above via an ellipsis in order to avoid them overlapping one another. Thanks!

Reply:
@bobsilverberg FWIW we've generally tried to avoid using truncation as a solution to long strings as it can make the remaining text hard to understand which then causes an accessibility issue. 

I think moving the right hand text underneath in some way with wrapping in extreme cases is probably preferable. If that's not great from a UX experience we might want to consider moving the secondary links into the footer, but that's definitely going to need UX input.

Reply:
Thanks @muffinresearch, I did recall we has an issue with ellipses before, so I'm glad I checked. 

@fantonangeli It looks like we need to take a different approach. Do you want to see if you can get it to wrap in a way that works?